datastore/additional_data/management/commands/rewrite_additional_data.py

```diff
@@ -64,9 +64,15 @@ def update_grants_page(page_num):
 
             for grant in page_grants:
```

Comment:
Is the plan to use `.select_related` on the query that generates the lists of grants?